### PR TITLE
Add list_order field to sort properties column

### PR DESCRIPTION
**Pull request by gbip**

### CODE REVIEW

**Comment by gbip** [comment]
@LePetitTim Ready for review.

Note that I have not included a test case. If you think that this feature needs one feel free to say so.

File: terra_geocrud/migrations/0067_crudviewproperty_list_order.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('terra_geocrud', '0066_auto_20210618_0901'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='crudviewproperty',
+            name='list_order',
```
Comment (jrmi) [inline terra_geocrud/migrations/0067_crudviewproperty_list_order.py:15]: Is `table_order` a better naming?

File: terra_geocrud/models.py [hunk 2]
```diff
@@ -272,6 +272,9 @@ class CrudViewProperty(models.Model):
     include_in_tile = models.BooleanField(default=False, db_index=True)
     required = models.BooleanField(default=False, db_index=True)
     order = models.PositiveSmallIntegerField(default=0, db_index=True)
+
+    """ This field is used to order properties in the list view """
```
Comment (jrmi) [inline terra_geocrud/models.py:276]: Table view?

**Comment by gbip** [comment]
@LePetitTim @jrmi Ready for review

**Review by jrmi — APPROVED** [review]

**Pull request merged**